pwnlib/util/nbstreamreader.py

```diff
@@ -0,0 +1,68 @@
+from threading import Thread
+from Queue import Queue, Empty
+
+class NonBlockingStreamReader:
```

Comment:
It looks like this file was lifted verbatim from http://eyalarubas.com/python-subproc-nonblock.html.  This work is copyrighted, and I don't see a license anywhere.  Please check with the author to see if he's okay with licensing the code to use under the MIT license